src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball randBall = new Ball();
+        randBall.color = new ColorSupplier().getRandomColor();
+        randBall.number = new Random().nextInt(100);
```

Comment:
Please extract new Random() and new ColorSupplier() to class variables.
Please also add all-args constructor for Ball, then you will be able make return new Ball(arg1, arg2)